libs/extended/src/System/Logger/Extended.hs

```diff
@@ -37,29 +37,25 @@ data LogFormat = JSON | Plain | Netstring
 -- | We use this as an intermediate structure to ease the implementation of the
 -- ToJSON instance but we could just inline everything. I think this has
 -- negligible impact and makes the code a bit more readable. Let me know
-data Element' = Element' [(Builder, Builder)] [Builder]
+data Element' = Element' Series [Builder]
```

Comment:
just curious: did you check that it's actually faster?  i always expect these things to be fused away by ghc, but i guess i'm expecting too much...

Reply:
no did not check.